styles_about.css

```diff
@@ -235,6 +235,30 @@ body {
   height: 45vw;
 }
 
+.expo1::after {
+  content: "";
+  display: block;
+  position: absolute;
+  top: 165vw;
+  left: 17.5vw;
+  width: 65vw;
+  height: 45vw;
+  background-color: #ec5242;
+  opacity: 0.5;
+}
+
+.expo2::after {
+  content: "";
+  display: block;
+  position: absolute;
+  top: 220vw;
+  left: 17.5vw;
+  width: 65vw;
+  height: 45vw;
+  background-color: #ec5242;
+  opacity: 0.5;
+}
```

Comment:
- [ ] You did a great job adding the orange layer on the image here as requested previously. But while doing so, I notice You have not aligned the boxes properly. Please remember we are expected to deliver something identical to the original design. You got this 👋
### Your design on desktop 
![image](https://user-images.githubusercontent.com/69077061/235282070-d6bb55b6-1fdd-4292-9a06-de268502557c.png)

### Your design on mobile 
![image](https://user-images.githubusercontent.com/69077061/235282082-7054fc95-a40f-42b1-a4f2-aefa1d85c8e2.png)